charts/stable/jetbrains-projector/values.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- See list of available images [here](https://github.com/JetBrains/projector-docker)
+  registry: registry.jetbrains.team/p/prj/containers
+  repository: projector-pycharm-c
+  pullPolicy: IfNotPresent
```

Comment:
Would it make sense to default this to `Always` since there is a `latest` tag set by default?

Reply:
Containers have been pushed so this is fixed with proper versioning